.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'Bug: Add_your_title_here'
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Also add 
```
<!--- Delete things that may be irrelevant to your bug report message. --->
```

Reply:
I have made the changes requested @Abhishek-kumar09